_posts/2016-04-11-growing-developers-at-greenfield.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Growing Developers at Greenfield"
+description: "Advice for junior developers getting started in their careers"
+category:
+tags: [lessons, advice, career]
+author: tom_chen
+---
+{% include JB/setup %}
+
+![tree]({{ BASE_PATH
+}}/assets/images/growing-developers-at-greenfield/tree.jpeg)
+
+Before I was a developer at Greenfield, I spent several years working in
+academia. After deciding to make the leap into full-time software development
+and completing [Launch Academy](https://www.launchacademy.com/) last year, I was
+anxious to join a company where I could work with and learn from experienced
+developers who were shipping high quality code. When I arrived at Greenfield, I
+was happy to find a team that was not only welcoming and supportive but also
```

Comment:
Dunno it kinda sounds weird to me when I change it around like that. Also, I googled it, and I think this is technically not the passive voice:

> In the active voice, the subject and verb relationship is straightforward: the subject is a be-er or a do-er and the verb moves the sentence along. In the passive voice, the subject of the sentence is neither a do-er or a be-er, but is acted upon by some other agent or by something unnamed

http://grammar.ccc.commnet.edu/grammar/passive.htm


I think in this case the Greenfield is the "be-er". Also the web design on that page is amazing.